kdl_parser/package.xml

```diff
@@ -20,18 +20,26 @@
   <url type="repository">https://github.com/ros/robot_model</url>
   <url type="bugtracker">https://github.com/ros/robot_model/issues</url>
 
-  <buildtool_depend version_gte="0.5.68">catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
 
   <build_depend version_gte="1.3.0">orocos_kdl</build_depend>
-  <build_depend>rosconsole</build_depend>
-  <build_depend>roscpp</build_depend>
   <build_depend>urdf</build_depend>
-  <build_depend>cmake_modules</build_depend>
-  <build_depend>rostest</build_depend>
+  <build_depend>urdfdom_headers</build_depend>
+  <build_depend>tinyxml</build_depend>
+  <build_depend>tinyxml_vendor</build_depend>
```

Comment:
dependencies should be listed in alphabetical order

Reply:
Fixed now.